AutomationHelpers/scripts/install.ps1

```diff
@@ -12,7 +12,7 @@ if (!$rxVersion)
     exit
 }
 $rxVersion = "RX$rxVersion"
-if ($rxVersion -ne "RX72" -and $rxVersion -ne "RX80")
+if ($rxVersion -ne "RX72" -and $rxVersion -ne "RX80" -and $rxVersion -ne "RX93")
```

Comment:
I would remove this check completely and always add the constant

Reply:
@ahoisl thanks for your comments, will update.